releasenotes/notes/expand-isomorphism-cfb646cfef66fa25.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    :func:`~retworkx.is_isomorphic` can take in a :class:`~retworkx.PyGraph`
+    or :class:`~retworkx.PyDiGraph` and two functions to compare node 
+    and edge data.
```

Comment:
We probably also should mention that `is_isomorphic_node_match` will work with PyGraph too now.
 
I also would add a second release note here mentioning that we take an optional node or edge matcher on is_isomorphic now.